go.mod

```diff
@@ -1,6 +1,8 @@
 module istio.io/istio
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.7
```

Comment:
Remove this. The go tooling adds it, but its a terrible idea and not what we want. 

While the tooling auto-adds it, if you remove it it will not add it back.

Reply:
> Remove this. The go tooling adds it, but its a terrible idea and not what we want.
> 
> While the tooling auto-adds it, if you remove it it will not add it back.

change it to 1.22.0 manually?

Reply:
Remove `toolchain`. You cannot change `go`, that is required to be 1.23.0 to support the packages we are importing.

Reply:
Gotcha, this branch has some issues. Created a new PR - https://github.com/istio/istio/pull/55594
Closing this one